submissions/onyemeMichael/htmlForms/signUp.html

```diff
@@ -75,11 +90,10 @@ <h3>Already got an account?<a href="#" class="log-in" >Sign in</a></h3>
             <p class="kt">Sign up for our Newsletter</p>
             <div class="newsletter">
                 <form>
-                    <input type="email" name="email" id="email" placeholder="Enter your email address" style="width: 50%; border-radius:0px; padding:5px">
+                    <input type="email" name="email" id="let" placeholder="Enter your email address" style="width: 50%; border-radius:0px; padding:5px">
```

Comment:
@MYKEL7 Move the style here to the css file

Reply:
Done.